src/polycubed/src/server/Resources/Endpoint/ParentResource.cpp

```diff
@@ -225,8 +225,8 @@ void ParentResource::del(const Request &request, ResponseWriter response) {
         std::vector<Response>{{ErrorTag::kNoContent, nullptr}},
         std::move(response));
     errors.push_back({ErrorTag::kCreated, nullptr});
-    if (auto d = core_->get_cubes_dump()) {
-      d->UpdateCubesConfig(request.resource(), nullptr, keys,
+    if (configuration::config.getCubesDumpEnabled()) {
+      core_->get_cubes_dump()->UpdateCubesConfig(request.resource(), nullptr, keys,
```

Comment:
Just a question: there was a loop before, with possible multiple cubes returned.
Why this loop has been removed here? Do we have always one cube?